### DISCUSSION TITLE
postal as an email gateway like promethus email gateway

### DISCUSSION THREAD

**Discussion by gitaayush** [postalserver/postal#1857] [Help with using Postal]
Hello all,

I have some question,  can we use postal as a email gateway like promethus email gateway, baracuda email gateway ??? 

I want to  use postal for only relay email server.

Thank you !!

**Comment by willpower232** [comment]
you could do this if you created a domain in postal for each one going through it and set up all your devices with credentials, I'm not sure how well it would work bounces etc
